helpFile.md

```diff
@@ -21,20 +21,20 @@ When should we go to Snowman's?
 1830
 1900
 ```
-Note we put our option on the same line as the command then on a new line we put our title.
+Note we put our option on the same line as the command then on a new line we put our question.
```

Comment:
This could be grammatically better:

> Take note of how we put any options on the same line as the command followed by the question on the next line.

or something